Automattic/woocommerce-payments
# rename "deposit" to "payout": instant deposit button & modal

Fixes #9542 

- Use "payout" in modal and button UI
- Update code comments
- Update component and type names
- Update CSS classes and relevant CSS rules
- Ensure existing automated tests pass

###### to do
~~**Work in progress** this is close but untested. My local dev site is not currently eligible for instant payouts.~~ Update: @nagpai [has tested full flow](https://github.com/Automattic/woocommerce-payments/pull/9554#issuecomment-2415728114), so I think we're good to go here.

- [x] Test all affected UI, make sure new wording works in context and nothing missed.
- [x] Ensure automated tests pass, and are testing the right things.

#### Changes proposed
- Update all instant-deposit related code and UI to use "payout" wording. 
- This PR aims to update everything that can be safely updated – excludes things like:
  - Documentation links / URLs
  - Database 
  - API 
  - custom hooks / wp.data



#### Testing instructions
###### your store needs to be eligible for instant deposits
- Account country must be `US`.
- Store/account currency must be `USD`.
- Add debit card or instant eligible account as payout method. Read https://woocommerce.com/document/woocommerce-payments/deposits/instant-deposits/#add-debit-card. Note – can now set up different test payout instruments in sandbox accounts in Stripe dashboard.
- Purchase USD$1000+ worth of products, using `4000000000000077` to skip pending period (speeds up TPV requirement).
- If account is new, trigger a manual deposit in Stripe dashboard, or hack eligibility rules in server ([has_account_completed_the_new_account_waiting_period](https://github.com/Automattic/woocommerce-payments-server/blob/1e529fd14eaeb7e9e852e363204143f07ed10f56/server/wp-content/rest-api-plugins/endpoints/wcpay/stripe/class-stripe-account.php#L746).

###### how to test this PR
- Ensure your store is eligible for instant deposits and has funds to deposit.
- Go to `Payments > Overview`.
- Instant payout button `Get $12 now` should appear on balances card.
- Click it and follow through the modal.
- Pay close attention and ensure there is no mention of `deposit` and all UI & copy is accurate and helpful 🌞

-------------------


**Post merge**
